devicetypes/smartthings/aeon-led-bulb.src/aeon-led-bulb.groovy

```diff
@@ -68,7 +68,7 @@ metadata {
 }
 
 def updated() {
-	response(refresh())
+	response(command(zwave.switchColorV3.switchColorSupportedGet()) + refresh())
```

Comment:
Why not put this in `refresh`?

Reply:
no reason to ever call it more than once

Reply:
For those of us (such as myself) who are less educated in the DTH world, what triggers `updated` and `installed`?

Reply:
`installed` is called exactly once, on install.

`updated` is called whenever device name or preferences are changed (counting the initial naming of the device)